package-sets/1.0.0.json

```
@@ -65,6 +65,21 @@
       "git": "https://github.com/purescm/purescript-unsafe-coerce.git",
       "ref": "20dbd7797e985b631459b8dbd4072cf3096ee1da",
       "dependencies": []
+    },
+    "safe-coerce": {
+      "git": "https://github.com/purescript/purescript-safe-coerce.git",
+      "ref": "7fa799ae80a38b8d948efcb52608e58e198b3da7",
+      "dependencies": [
+        "unsafe-coerce"
+      ]
+    },
+    "newtype": {
+      "git": "https://github.com/purescript/purescript-newtype.git",
+      "ref": "0a0563efca18fe6b2e9d4cc0b8574e507dee4453",
+      "dependencies": [
+        "prelude",
+        "safe-coerce"
+      ]
```

We should be able to just pull upstream things from the registry:
```suggestion
    },
    "safe-coerce": "2.0.0",
    "newtype": "5.0.0"
```